app/src/src/components/common/FormatDate/MonthDayYear.jsx

```diff
@@ -0,0 +1,27 @@
+import YYYYMMDD from '../FormatDate/YYYYMMDD';
+// takes in Date and returns the date in Month Day Year format (ex: January 1 1960)
+const MonthDayYear = ({ date }) => {
```

Comment:
We should probably give this a nicer name, like `toMonthDayYear`. Also, functions that are not react components in javascript should be lowercased.